api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -97,6 +90,3 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${CATALOG_CODE} java \
     -Djava.protocol.handler.pkgs=com.ibm.crypto.provider \
     -Dloader.path=${COMMON_LIB} \
     -jar "${JAR_FILE}" &
-pid=$!
-echo "pid=${pid}"
```

Comment:
I am afraid that our Integration Tests are depending on this echo of PID to get PID of started process. I will try to look for alternatives so we can remove this.. @jackjia-ibm what is the motivation behind removing this echo?

Reply:
It does not seem that there is some easy solution to get the pid that would work reasonably well across platforms apart from reading pid from the stdout